content/W2016/W16-event-airbnb.md

```diff
@@ -0,0 +1,29 @@
+Title: Escape Room with Airbnb
+Date: 2016-02-15
+Category: Events
+Tags: social
+Slug: airbnb-escape-w16
+Author: Fatema Boxwala
+Summary: Hang out with some cool people from Airbnb while trying to escape
+some rooms!
+
+Have you been wondering what those “escape room” things everyone’s been talking
+about lately are? Come find out with WiCS and Airbnb!
+
+We’re going out to Adventure Rooms Canada to try it for ourselves, and maybe
+beat a record while we’re there.
+
+This event has limited attendance so [register soon](https://docs.google.com/forms/d/1v7OqMeQ0QI6RHETIIEA7VgoUeUm3VT7m0QbP0fPUtQM/viewform)!
+
+Trigger warning: this event involves some activities that might be disturbing
```

Comment:
I prefer "content warning" to trigger warning, as the former wording is more inclusive.